lib/parse-client-options.js

```diff
@@ -7,6 +7,7 @@ const getRequestAgent = require('./get-request-agent')
 const DEFAULTS = require('./defaults')
 const OPTION_NAMES = [
   'timeout',
+  'baseUrl',
```

Comment:
remove 

```js
   'host',
   'pathPrefix',
   'protocol',
   'port',
```

instead we should check for them "manually" below. That way it will be easier to remove them in future, as they are implemented in only one block

Reply:
Sure.